src/main/cpp/chassis/pose/DragonVisionPoseEstimator.cpp

```diff
@@ -13,20 +13,5 @@
 // OR OTHER DEALINGS IN THE SOFTWARE.
 //====================================================================================================================================================
 
-#include "chassis/definitions/ChassisConfig.h"
-#include "chassis/definitions/ChassisConfigMgr.h"
-#include "chassis/pose/DragonSwervePoseEstimator.h"
 #include "chassis/pose/DragonVisionPoseEstimator.h"
-
-DragonVisionPoseEstimator::DragonVisionPoseEstimator()
-{
-    auto chassis = ChassisConfigMgr::GetInstance()->GetCurrentChassis();
-    if (chassis != nullptr)
-    {
-        auto poseEst = chassis->GetSwervePoseEstimator();
-        if (poseEst != nullptr)
-        {
-            poseEst->RegisterVisionPoseEstimator(this);
-        }
-    }
-}
+DragonVisionPoseEstimator::DragonVisionPoseEstimator() = default;
```

Comment:
Why is this here and not in the .h file?

What is the purpose of this file anymore?

Reply:
I did try a clean build too

Reply:
Did you put 

DragonVisionPoseEstimator() = default;

in the .h file?

Reply:
make dragonvisionposeestimator a singleton

Reply:
Done